docker-compose.yml

```diff
@@ -20,7 +20,7 @@ services:
       - mysql
       - db_phpunit
     ports:
-      - "80:80"
+      - "8080:80"
```

Comment:
Could we keep using port `80` for consistency with the developer docs? Currently they're referencing `stream.local` without the port number.

Reply:
My bad, must have forgot to change it back. I'll be sure to fix it when I rebase